amit/html/Assigment3/Amit3_1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+ <html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta hhtp-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Lists</title>
+   </head>
+   <body>
+       <div>
+          <h1>AmberSoft</h1>
+            <ol style="list-style-type: number">MI-MEAN
+               <ol style="list-style-type: lower-roman">
+                 <li>MI-MEAN-6</li>
+                 <ul style= "list-style-type: disc">
+                    <li> Shweta </li> 
+                    <li> Ajay </li>
+                    <li> Trupti </li>
+                 </ul>
+                 <li>MI-MEAN-7</li>                
+                 <ul style= "list-style-type: disc">
+                    <li> Sagar </li> 
+                    <li> Umesh </li>
+                 </ul>
+                 <li>MI-MEAN-8</li>        
+                 <ul style= "list-style-type: disc">
+                     <li> Mihika </li> 
+                     <li> Pallavi </li>
+                     <li> Reva </li> 
+                 </ul>
+                 <li>MI-MEAN-9</li>   
+                 <ul style= "list-style-type: disc">
+                     <li> Anil </li> 
+                     <li> Manoj</li>         
+                  </ul>   
+                </ol>
+                <li> System </li>
+                  <ol style="list-style-type: lower-roman">
+                     <li> System 1 </li>
+                      <ul style= "list-style-type: disc">
+                        <li> Anu </li> 
+                        <li> Dipti </li>   
+                      </ul>  
+                     <li> System 2 </li>    
+                     <ul style= "list-style-type: disc">
+                        <li> Triveni </li> 
+                        <li>Kalyani </li>  
+                     </ul> 
+                   </ol>
+                </li>
+            </ol>      
+        </div> 
+     </body>
+ </html>
+                 
```

Comment:
remove extra lines only one line space expected

Reply:
sir told 1 line is nessory after complte code